contracts/deposit-service/AxelarDepositService.sol

```diff
@@ -255,15 +260,12 @@ contract AxelarDepositService is Upgradable, DepositServiceBase, IAxelarDepositS
     ) external {
         if (msg.sender != refundIssuer) revert NotRefundIssuer();
         if (receiver == address(0)) revert InvalidAddress();
+        if (amount == 0) revert InvalidAmount();
 
         if (token == address(0)) {
-            if (amount == 0) revert InvalidAmount();
-
-            (bool sent, ) = receiver.call{ value: amount }('');
-
-            if (!sent) revert NativeTransferFailed();
+            receiver.safeNativeTransfer(amount);
```

Comment:
might need to allow passing in more gas on some chains, where it naturally costs more than 2300. We can add that ability if needed